config/ignition_citadel_gazebo11_ubuntu_focal.yaml

```diff
@@ -2,136 +2,144 @@ name: ignition_citadel_gazebo11_ubuntu_focal
-Package (% =libignition-sensors3-altimeter) |\
-Package (% =libignition-sensors3-altimeter-dev) |\
-Package (% =libignition-sensors3-camera) |\
-Package (% =libignition-sensors3-camera-dev) |\
-Package (% =libignition-sensors3-core-dev) |\
-Package (% =libignition-sensors3-depth-camera) |\
-Package (% =libignition-sensors3-depth-camera-dev) |\
-Package (% =libignition-sensors3-dev) |\
-Package (% =libignition-sensors3-gpu-lidar) |\
-Package (% =libignition-sensors3-gpu-lidar-dev) |\
-Package (% =libignition-sensors3-imu) |\
-Package (% =libignition-sensors3-imu-dev) |\
-Package (% =libignition-sensors3-lidar) |\
-Package (% =libignition-sensors3-lidar-dev) |\
-Package (% =libignition-sensors3-logical-camera) |\
-Package (% =libignition-sensors3-logical-camera-dev) |\
-Package (% =libignition-sensors3-magnetometer) |\
-Package (% =libignition-sensors3-magnetometer-dev) |\
-Package (% =libignition-sensors3-rendering) |\
-Package (% =libignition-sensors3-rendering-dev) |\
-Package (% =libignition-sensors3-rgbd-camera) |\
-Package (% =libignition-sensors3-rgbd-camera-dev) |\
-Package (% =libignition-sensors3-thermal-camera) |\
-Package (% =libignition-sensors3-thermal-camera-dev)), \
-$Version (% 3.3.0-1~*)) |\
-((Package (% =ignition-tools) |\
-Package (% =libignition-tools-dev)), \
-$Version (% 1.4.1-1~*)) |\
-((Package (% =ignition-transport8) |\
-Package (% =libignition-transport8) |\
-Package (% =libignition-transport8-core-dev) |\
-Package (% =libignition-transport8-dbg) |\
-Package (% =libignition-transport8-dev) |\
-Package (% =libignition-transport8-log) |\
-Package (% =libignition-transport8-log-dev)), \
-$Version (% 8.2.1-1~*)) |\
-((Package (% =ogre-2.1) |\
-Package (% =libogre-2.1) |\
-Package (% =libogre-2.1-dev)),
-$Version (% 2.0.9999~20180616~06a386f-3osrf~*)) |\
-((Package (% =sdformat9) |\
-Package (% =libsdformat9) |\
-Package (% =libsdformat9-dbg) |\
-Package (% =libsdformat9-dev) |\
-Package (% =sdformat9-doc) |\
-Package (% =sdformat9-sdf)), \
-$Version (% 9.7.0-1~*)) \
+(Package (= gazebo11) |\
+ Package (= gazebo11-common) |\
+ Package (= gazebo11-dbg) |\
+ Package (= gazebo11-doc) |\
+ Package (= gazebo11-plugin-base) |\
+ Package (= libgazebo11) |\
+ Package (= libgazebo11-dev) \
+), $Version (% 11.10.2-1*) |\
+(Package (= ignition-citadel) \
+), $Version (% 1.0.2-1*) |\
+(Package (= ignition-cmake2) |\
+ Package (= libignition-cmake2-dev) \
+), $Version (% 2.12.1-1*) |\
+(Package (= ignition-common4) |\
+ Package (= libignition-common4) |\
+ Package (= libignition-common4-av) |\
+ Package (= libignition-common4-av-dev) |\
+ Package (= libignition-common4-core-dev) |\
+ Package (= libignition-common4-dev) |\
+ Package (= libignition-common4-events) |\
+ Package (= libignition-common4-events-dev) |\
+ Package (= libignition-common4-graphics) |\
+ Package (= libignition-common4-graphics-dev) |\
+ Package (= libignition-common4-profiler) |\
+ Package (= libignition-common4-profiler-dev) \
+), $Version (% 4.5.0-1*) |\
```

Comment:
Citadel uses `common3`. I haven't looked at the script to see where you're getting the versions from, but [gazebodistro](https://github.com/ignition-tooling/gazebodistro/blob/master/collection-citadel.yaml) is a good resource

Reply:
Yeah, I lost track of which collections I had actually vetted and which were just copy-paste jobs. I've updated the script to use the versions described in that project (although it's still hard-coded so changes in the gazebodistro repo won't be picked up automatically)